release_tools/README.adoc

```diff
@@ -0,0 +1,51 @@
+= How to perform a semi-automatic release =
+
+== Prerequisities ==
+
+* The `.env` file.
+  Create `.env` file in this directory (i.e. where the release scripts are).
+  You are supposed to define environment variables that contain sensitive information, s.a. GitHub access tokens.
+  Exact contents of the file are described below.
+
+* Github automation
+ * System with `python3.6` or greater, installed [PyGithub](https://pypi.python.org/pypi/PyGithub/1.35) module.
```

Comment:
Just a minor nitpick, this is a wrong format of a link. According to http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links it should look like
`http://asciidoctor.org[Asciidoctor]`